subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -558,7 +558,13 @@ uint8_t ll_req_peer_sca(uint16_t handle)
 		return BT_HCI_ERR_UNKNOWN_CONN_ID;
 	}
 
+#if !defined(CONFIG_BT_LL_SW_LLCP_LEGACY)
```

Comment:
Should maybe BT_CTLR_SCA_UPDATE depend on !BT_LL_SW_LLCP_LEGACY to avoid this configuration in the first place?

Reply:
True.